### PR TITLE
[RISCV] Add extract_subvector tests for a statically-known VLEN. NFC

### PR DESCRIPTION
**Pull request by lukel97**
This is partly a precommit for an upcoming patch, and partly to remove the fixed length LMUL restriction similarly to what was done in https://reviews.llvm.org/D158270, since it's no longer that relevant.

### CODE REVIEW

File: llvm/test/CodeGen/RISCV/rvv/fixed-vectors-extract-subvector.ll [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
-; RUN: llc -mtriple=riscv64 -mattr=+m,+v -riscv-v-vector-bits-min=128 -riscv-v-fixed-length-vector-lmul-max=2 -verify-machineinstrs < %s | FileCheck %s --check-prefixes=CHECK,LMULMAX2
-; RUN: llc -mtriple=riscv64 -mattr=+m,+v -riscv-v-vector-bits-min=128 -riscv-v-fixed-length-vector-lmul-max=1 -verify-machineinstrs < %s | FileCheck %s --check-prefixes=CHECK,LMULMAX1
+; RUN: llc -mtriple=riscv64 -mattr=+m,+v -riscv-v-vector-bits-min=128 -verify-machineinstrs < %s | FileCheck %s --check-prefixes=CHECK,UNKNOWNVLEN
```
Comment (preames) [inline llvm/test/CodeGen/RISCV/rvv/fixed-vectors-extract-subvector.ll:2]: Can you remove -riscv-v-vector-bits-min=128 from the first line here?  It's implied by V.

File: llvm/test/CodeGen/RISCV/rvv/fixed-vectors-extract-subvector.ll [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+; RUN: llc -mtriple=riscv64 -mattr=+m,+v -riscv-v-vector-bits-min=128 -riscv-v-vector-bits-max=128 -verify-machineinstrs < %s | FileCheck %s --check-prefixes=CHECK,KNOWNVLEN
```
Comment (preames) [inline llvm/test/CodeGen/RISCV/rvv/fixed-vectors-extract-subvector.ll:3]: I'd suggest CHECK-V and CHECK-KNOWNVLEN128

**Review by preames — APPROVED** [review]
LGTM w/comments addressed.

File: llvm/test/CodeGen/RISCV/rvv/fixed-vectors-extract-subvector.ll (hunk 1 above)
Comment (lukel97) [reply llvm/test/CodeGen/RISCV/rvv/fixed-vectors-extract-subvector.ll:2]: Yeah that always bothered me too, it's in a lot of other tests. I can do a big sweep and remove them in a separate PR?

**Pull request merged**